posthog/models/filters/mixins/property.py

```diff
@@ -22,14 +22,17 @@ def properties(self) -> List[Property]:
         else:
             loaded_props = _props
 
-        return self._parse_properties(loaded_props)
+        # if grouped properties
+        if isinstance(loaded_props, dict) and "type" in loaded_props and "groups" in loaded_props:
+            # property_groups is main function from now on
+            return []
+        else:
+            # old style dict properties or a list of properties
+            return self._parse_properties(loaded_props)
 
     @cached_property
     def property_groups(self) -> PropertyGroup:
-        _props = self._data.get(PROPERTY_GROUPS, None)
-
-        if not _props:
-            return PropertyGroup(type=PropertyOperatorType.AND, groups=self.properties)
+        _props = self._data.get(PROPERTIES)
```

Comment:
Keeping the API the same^^

and at the same time, parsing such that `property_groups` are the new properties. Eventually, we should remove all occurences of properties, to be replaced with property_groups.